php/classes/integrations/elementor/class-template-importer.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace SeriouslySimplePodcasting\Integrations\Elementor;
+
+use Elementor\TemplateLibrary\Manager;
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Template Importer for Elementor
+ *
+ * @author      Jonathan Bossenger
+ * @category    Class
+ * @package     SeriouslySimplePodcasting/Classes
+ * @since       1.19.18
+ */
+class Template_Importer {
+
+	protected $template_name = 'ssp-elementor-template-2020-09-30.json';
```

Comment:
We should update the importer to scan the template_path directory (see comment below)